server/handler/handler_test.go

```diff
@@ -317,6 +317,21 @@ func TestSessionHelper(t *testing.T) {
 	assert.Equal(t, http.StatusOK, rec.Result().StatusCode, "Expected OK from auth with cookie.")
 }
 
+func TestTimeouts(t *testing.T) {
```

Comment:
Minor comment, let's name the test `TestGetEngineConfigurationAPI`.

Reply:
Oh yeah I saw that comment when I was away.  For sure, I changed the name everywhere but missed the test case.